src/Raven.TestDriver/RavenServerRunner.cs

```diff
@@ -34,21 +34,22 @@ private static ProcessStartInfo GetProcessStartInfo(TLocator locator)
 
             var commandArguments = new List<string>
             {
-                    locator.CommandArguments,
-                    "--ServerUrl=http://127.0.0.1:0",
-                    "--RunInMemory=true",
-                    "--Testing.ParentProcessId=" + Process.GetCurrentProcess().Id,
-                    "--Setup.Mode=None"
-                };
+                locator.CommandArguments,
+                "--ServerUrl=http://127.0.0.1:0",
+                "--RunInMemory=true",
+                "--Testing.ParentProcessId=" + Process.GetCurrentProcess().Id,
+                "--Setup.Mode=None"
+            };
 
             var argumentsString = string.Join(" ", commandArguments); 
-            return new ProcessStartInfo()
+            return new ProcessStartInfo
             {
                 FileName = locator.Command,
                 Arguments = argumentsString,
                 CreateNoWindow = true,
                 RedirectStandardOutput = true,
                 RedirectStandardError = true,
+                UseShellExecute = false,
```

Comment:
Isn't this the default already?

Reply:
Also test on Linux, please.

Reply:
No, the default is `true`. And the user confirmed that this fixed his issue.